C-Plus-Plus/cp/busy_man.cpp

```diff
@@ -0,0 +1,65 @@
+/*Problem Statement:
+You are actually very busy man. You have a big schedule of activities. 
+Your aim is to do as much as activities as possible.
+It is given that, if you go to date with crush, 
+you cannot participate in the coding contest and you can’t watch the movie. 
+Also if you play DotA, you can’t study for the exam. 
+If you study for the exam you can’t sleep peacefully. 
+The maximum number of activities that you can do for this schedule is 3. */
+
+#include <bits/stdc++.h>
+using namespace std;
+
+bool compare(pair<int, int> p1, pair<int, int> p2)
+{
+    return p1.second < p2.second;
+}
+
+int main()
+{
+    int t, n, s, e;
+    cin >> t;
```

Comment:
Use proper naming conventions don't use single letter and also while reading inputs use cout for display proper message

What is `t`? Is it used for reading testcase then remove it as we don't have online judge for testcases.